index.js

```diff
@@ -27,7 +27,8 @@ exports.register = (server, options, next) => {
   if (!pluginOptions.envs[pluginOptions.env]) {
     pluginOptions.env = '*';
   }
-  let robotText = _.reduce(pluginOptions.envs[options.env], (memo, disallowList, userAgent) => {
+  // by default, production should return blank:
+  let robotText = options.env === 'production' ? '' : _.reduce(pluginOptions.envs[options.env], (memo, disallowList, userAgent) => {
```

Comment:
so prod default config should output nothing. but if I set disallows, it should output something